pandas/tests/groupby/test_categorical.py

```diff
@@ -963,3 +963,67 @@ def test_shift(fill_value):
                               categories=['a', 'b', 'c', 'd'], ordered=False)
     res = ct.shift(1, fill_value=fill_value)
     assert_equal(res, expected)
+
+
+def test_groupby_series_observed():
+    # GH 24880
+    df = DataFrame({'a': ['x', 'x', 'x', 'y'],
+                    'b': ['a', 'a', 'b', 'a'],
+                    'c': [1, 2, 3, 4]})
+    df['a'] = df['a'].astype('category')
+    df['b'] = df['b'].astype('category')
+
+    # test .agg and .apply when observed == False
```

Comment:
Rather than duplicate things in individual sections here should try and parametrize on agg/apply and True/False (for observed)